src/platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -223,3 +116,63 @@ export SENTRY_URL=https://mysentry.invalid/
+
+</Alert>
+
+Setting up an App Store Connect custom repository requires an App Store Connect API Key.
+
+Follow the documentation on [Creating API Keys for App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api) and provide the _Issuer_, _Key ID_, and _Private Key_. The API Key needs to have the _Developer_ role in order to fetch build details.
+
+**Invalid Credentials**
+
+If your App Store Connect API key is revoked on the Apple side, the Sentry custom repository using this key's credentials will no longer be able to discover and fetch dSYM files and the following warnings will be displayed in Sentry's User Interface:
+
+#### `Issue Details`
+
+![Warning to update credentials in an issue's details](issue-details-warning.png)
+
+#### `Project Settings`
+
+![Warning to update credentials in sidebar](sidebar-warning.png)
+
+#### `Custom Repositories`
+
+![Warning to update credentials in Custom Repositories](custom-repositories-warning.png)
+
+When using Fastlane's _upload_to_testflight_ action, it is recommended to pass the _skip_waiting_for_build_processing_ parameter to speed up the action.
+
+## Bitcode {#dsym-with-bitcode}
+
+If [Bitcode](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AppThinning/AppThinning.html#//apple_ref/doc/uid/TP40012582-CH35-SW2), no longer available [starting with Xcode 14](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes), **is enabled** in your project, you need to download the dSYMs from App Store Connect after it finishes processing your build, and then upload them to Sentry.
+You can achieve this by using either:
+
+- Fastlane's [download_dsyms](https://docs.fastlane.tools/actions/download_dsyms/) action and then upload the dSYMs with the [Sentry Fastlane plugin](#fastlane).
+- [Sentry App Store Connect integration](#appstore-connect), which fetches the dSYMS direclty from App Store Connect.
+- Manually from App Store Connect:
+  - Open Xcode Organizer, go to your app, and click _Download dSYMs..._
+  - Login to App Store Connect, go to your app, go to “Activity", click the build number to go into the detail page, and click _Download dSYM_.
+  - Manually upload the dSYMs with [sentry-cli](#sentry-cli).
+
+These dSYMs contain obfuscated symbol names and paths. You must upload the BCSymbolMap file stored in your xcarchive so Sentry can  properly symbolicate your crash reports. You can do this by using the upload-dif command of either [sentry-cli](#sentry-cli) or the [Sentry Fastlane plugin](#fastlane), which will automatically find and upload all BCSymbolMap files, given the path to the app's xcarchive.
```

Comment:
```suggestion
These dSYMs contain obfuscated symbol names and paths. You must upload the BCSymbolMap file stored in your xcarchive so Sentry can properly symbolicate your crash reports. You can do this by using the upload-dif command of either [sentry-cli](#sentry-cli) or the [Sentry Fastlane plugin](#fastlane), which will automatically find and upload all BCSymbolMap files once it has the path to the app's xcarchive.
```
Want to make sure that the update I made to the last line makes sense? Am I understanding what you meant by "given the path to the app's...?" correctly?

Reply:
I tried it differently. Please have another look.